storage/debugbar/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Please add debugbar to composer manifest.

Reply:
Added :D